pkgs/by-name/gi/git-secret/package.nix

```diff
@@ -6,40 +6,49 @@
   git,
   gnupg,
   gawk,
+  installShellFiles,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "git-secret";
-  version = "0.5.0";
+  version = "0.5.0-unstable-09-12-2024";
```

Comment:
According to the [guidelines](https://github.com/NixOS/nixpkgs/tree/master/pkgs#versioning), the current date in the new version should be in the reversed format `yyyy-mm-dd`.

Reply:
oh okay, I missed that. Updating it.